### DISCUSSION TITLE
老是无法正常链接，不能正常使用

### DISCUSSION THREAD

**Discussion by Cicichen888** [getlantern/lantern-forum-cn#554] [Q&A]
手机端盒电脑端都是经常无法链接，不能正常使用，又联系不到客服，真的很懊糟呢！
不行的话我也要退款啊！！！
还有，能不能让自己选节点呢？？？出钱的为什么不能自选，有时候需要选节点

**Comment by MaxMKMax** [comment]
@Cicichen888 Hi CiCi Chen 您好. 很抱歉您没得到客服的回复. 以后您有什么事直接在这里找我 , 我负责管理GitHub的. 

请问您有更新最新版本吗? 请查看. 
https://github.com/getlantern/lantern

https://lantern.io/zh-Hans/faq
如果还遇到什么问题, 您可以参考上面的链接. 谢谢您.

您帮我提交各应用内报告吧. 我来让技术团队帮您查看什么问题. 提交报告后请给我 8 位数票号. 

选节点这件事情, 技术团队在搞着了. 估计一个月后就能上线了.

您再给我们 2 个星期的机会好吗? 如果真不行, 我再帮您申请退款. 谢谢您的体谅!
